notebooks/1_spikesorting.ipynb

```diff
@@ -4,26 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<p>I think this step creates a workspace and adds the recording to it. kachery is involved but it isn't the main point - this is for manual curation later on. I think the workspace stuff needs to be explained here. I can do it if you don't feel comfortable with it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/nwb_datajoint/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>That would be amazing! I think my understanding isn't comprehensive enough to do it justice. </p>